index.html

```diff
@@ -31,6 +31,100 @@
+                    <li><a href=""><button id="skill">javaScript</button></a></li>
+                </ul>
+                <button class="normal-button">See Project</button>
+            </div>
+
+
+        </div>
+        <div class="card-works">
+            <img class="porfolio-card1" src="./images/Snapshoot Portfolio 2.png" alt="">
+        <div class="project-desc">
+            <p class="project-title">Multi-Post Stories</p>
+            <div class="frame2">
+                <h3>CANOPY</h3>
+                <img src="./images/Counter.png" alt="">
+                <h4>Back End Dev</h4>
+                <img src="./images/Counter.png" alt="">
+                <h4>2015</h4> 
+            </div>
+             
+            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tag">
+                <li><a href=""><button id="skill">html</button></a></li>
+                <li><a href=""><button id="skill">css</button></a></li>
+                <li><a href=""><button id="skill">javaScript</button></a></li>
+            </ul>
+            <button class="normal-button">See Project</button>
+        </div>
+
+
+    </div>
+    <div class="card-works">
+        <img class="porfolio-card1" src="./images/Snapshoot Portfolio 3.png" alt="">
+    <div class="project-desc">
+        <p class="project-title">Tonic</p>
+        <div class="frame2">
+            <h3>CANOPY</h3>
+            <img src="./images/Counter.png" alt="">
+            <h4>Back End Dev</h4>
+            <img src="./images/Counter.png" alt="">
+            <h4>2015</h4> 
+        </div>
+         
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag">
+            <li><a href=""><button id="skill">html</button></a></li>
+            <li><a href=""><button id="skill">css</button></a></li>
+            <li><a href=""><button id="skill">javaScript</button></a></li>
+        </ul>
+        <button class="normal-button">See Project</button>
+    </div>
+
+
+</div>
+<div class="card-works">
+    <img class="porfolio-card1" src="./images/Snapshoot Portfolio 4.png" alt="">
+<div class="project-desc">
+    <p class="project-title">Multi-Post Stories</p>
+    <div class="frame2">
+        <h3>CANOPY</h3>
+        <img src="./images/Counter.png" alt="">
+        <h4>Back End Dev</h4>
+        <img src="./images/Counter.png" alt="">
+        <h4>2015</h4> 
+    </div>
+     
+    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tag">
+        <li><a href=""><button id="skill">html</button></a></li>
+        <li><a href=""><button id="skill">css</button></a></li>
+        <li><a href=""><button id="skill">javaScript</button></a></li>
+    </ul>
+    <button class="normal-button">See Project</button>
+</div>
+
+
+</div>
+    </section>
```

Comment:
Nice work building the `work` section of the project, But I think you may have forgotten the `about` section😉 we are also [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to build the `about` section😉😊

<img width="611" alt="THE ABOUT MYSELF SECTION" src="https://user-images.githubusercontent.com/64643995/220666266-4e07650f-a7dc-4138-893a-b69505548fb0.png">

kindly go through the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and add this section to your project😉

Reply:
@Ibizugbe
I would like to thank you for reviewing my project. 
 I also would like to know if  the contact us section included?

Reply:
@Ibizugbe I added the About section